src/data/CharacterInfo.ts

```diff
@@ -0,0 +1,170 @@
+export interface CharacterInfoModel {
+    id: string;
+    charName: string,
+    // if the charName's first name is not equal to image location
+    imageAlias?: string;
+    age: string,
+    birthday: string,
+    urobium: string,
+    interests: string,
+    occupation: string,
+    biography: string,
+    color: string
+}
+
+// from: https://www.reddit.com/r/react/comments/1aitdhu/comment/koxuq75/
+// we need to get images during build time, but we can try to at least map what's their path
+// with the helper function below
+const allImages: string[] = Object.values(import.meta.glob('@/assets/images/chargal/*.png',
+    {eager: true, import: 'default'},
+));
+
+/**
+ * Get equivalent image for given character
+ * @param charInfo
+ * @return valid .png image stored in chargal. it does not accept *.jpg unless you set it up
+ * see https://vite.dev/guide/features#glob-import
+ */
+export const GetImage = (charInfo: CharacterInfoModel) => {
+    return allImages.find(img => img.includes(
+        charInfo.imageAlias
+        ?? charInfo.charName.split(" ")[0].toLowerCase())
+    );
+};
```

Comment:
This is where we dynamically figure out the path to the image related to a character